garage/tf/core/networks.py

```diff
@@ -0,0 +1,48 @@
+import tensorflow as tf
+
+
+def mlp(input_var,
+        output_dim,
+        hidden_sizes,
+        hidden_nonlinearity=tf.nn.relu,
+        output_nonlinearity=None,
+        name="mlp",
+        hidden_w_init=tf.contrib.layers.xavier_initializer(),
+        hidden_b_init=tf.zeros_initializer(),
+        output_w_init=tf.contrib.layers.xavier_initializer(),
+        output_b_init=tf.zeros_initializer(),
+        batch_normalization=False,
+        weight_normalization=False,
+        is_training=True):
+
+    with tf.variable_scope(name):
+        l_hid = input_var
+        if batch_normalization:
+            l_hid = tf.layers.batch_normalization(l_hid, training=is_training)
+        for idx, hidden_size in enumerate(hidden_sizes):
+            l_hid = tf.layers.dense(
+                inputs=l_hid,
+                units=hidden_size,
+                activation=hidden_nonlinearity,
+                kernel_initializer=hidden_w_init,
+                bias_initializer=hidden_b_init,
+                name="hidden_%d" % idx,
```

Comment:
please use Python 3 `.format()` syntax. `%` syntax is considered deprecated in Python 3.